x/ccv/provider/keeper/relay.go

```diff
@@ -358,28 +311,36 @@ func (k Keeper) OnRecvSlashPacket(ctx sdk.Context, packet channeltypes.Packet, d
 		return channeltypes.NewResultAcknowledgement(ccv.V1Result)
 	}
 
-	// Queue a slash entry to the global queue, which will be seen by the throttling logic
-	k.QueueGlobalSlashEntry(ctx, providertypes.NewGlobalSlashEntry(
-		ctx.BlockTime(),   // recv time
-		chainID,           // consumer chain id that sent the packet
-		packet.Sequence,   // IBC sequence number of the packet
-		providerConsAddr)) // Provider consensus address of val to be slashed
-
-	// Queue slash packet data in the same (consumer chain specific) queue as vsc matured packet data,
-	// to enforce order of handling between the two packet data types.
-	if err := k.QueueThrottledSlashPacketData(ctx, chainID, packet.Sequence, data); err != nil {
-		return ccv.NewErrorAcknowledgementWithLog(ctx, fmt.Errorf("failed to queue slash packet data: %s", err.Error()))
+	meter := k.GetSlashMeter(ctx)
```

Comment:
Meat of provider changes is here. If slash meter is negative, bounce packet, otherwise decrement meter and handle packet